client/src/utils/sortFunctions.js

```diff
@@ -0,0 +1,11 @@
+export function sortByName(array) {
+  const sortedArray = [...array].sort((a, b) =>
+    a.name.toLowerCase() < b.name.toLowerCase() ? -1 : 1
```

Comment:
```suggestion
    a.name.toLowerCase().localCompare(b.name.toLowerCase())
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare